hacking/aws_config/testing-iam-policy.json.j2

```diff
@@ -149,7 +149,7 @@
                 "rds:DeleteDBInstance"
             ],
             "Resource": [
-                "arn:aws:rds:{{aws_region}}:{{aws_account}}:db:ansible-testing*"
+                "arn:aws:rds:{{aws_region}}:{{aws_account}}:db:ansible-test*"
```

Comment:
resource_prefix resolves to ansible-testing- (and plus some other stuff running under docker). This change will break everything relying on resource_prefix.